src/main/java/com/powsybl/openloadflow/util/Reports.java

```diff
@@ -195,4 +196,74 @@ public static Reporter createPostContingencySimulation(Reporter reporter, String
         return reporter.createSubReporter("postContingencySimulation", "Post-contingency simulation '${contingencyId}'",
                 "contingencyId", contingencyId);
     }
+
+    public static Reporter createNewtonRaphsonReporter(Reporter reporter, int networkNumCc, int networkNumSc) {
+        return reporter.createSubReporter("newtonRaphson", "Network CC${networkNumCc} SC${networkNumSc}",
```

Comment:
Maybe we can put something more explicit here Like "Newton Raphson on Network CC#X SC#X".